orcoursetrion/tests/test_cmd.py

```diff
@@ -49,5 +49,57 @@ def test_cmd_create_xml_repo(self):
                     self.TEST_COURSE,
                     self.TEST_TERM,
                     self.TEST_TEAM,
+                    None,
                     self.TEST_DESCRIPTION
                 )
+        args = [
+            'orcoursetrion', 'create_xml_repo',
+            '-c', self.TEST_COURSE,
+            '-t', self.TEST_TERM,
+            '-d', self.TEST_DESCRIPTION,
+            '-m', 'archlight', 'dreadnought',
+            '-g', self.TEST_TEAM,
+        ]
+        with mock.patch('sys.argv', args):
+            with mock.patch('orcoursetrion.cmd.actions') as mocked_actions:
+                execute()
+                self.assertTrue(mocked_actions.create_xml_repo.called)
+                mocked_actions.create_xml_repo.assert_called_with(
+                    self.TEST_COURSE,
+                    self.TEST_TERM,
+                    self.TEST_TEAM,
+                    ['archlight', 'dreadnought'],
+                    self.TEST_DESCRIPTION
+                )
+
+    def test_cmd_put_team(self):
+        """
+        Command line test of create_export_repo
+        """
+        args = [
+            'orcoursetrion', 'put_team',
+            '-o', self.ORG,
+            '-g', self.TEST_TEAM,
+        ]
+        with mock.patch('sys.argv', args):
+            with mock.patch('orcoursetrion.cmd.actions') as mocked_actions:
+                execute()
+                self.assertTrue(mocked_actions.put_team.called)
+                mocked_actions.put_team.assert_called_with(
+                    self.ORG, self.TEST_TEAM, False, None
+                )
+
+        args = [
+            'orcoursetrion', 'put_team',
+            '-o', self.ORG,
+            '-r',
+            '-g', self.TEST_TEAM,
+            '-m', 'bizarnage', 'chemistro',
```

Comment:
the assertion below would prove otherwise, or at least assert it.  Using nargs=* you can add 0 or more arguments to an option, so here I'm adding 2.